src/style.css

```diff
@@ -0,0 +1,7 @@
+.o2xp-to-print-columns{
+    color: rgba(0, 0, 0, 0.54);
+}
+
+.MuiDialog-paperWidthXl{
```

Comment:
Don't overide material-ui css directly. If user has material ui as dépendance you will overwrite his css too